src/topology/separation.lean

```diff
@@ -738,3 +736,48 @@ begin
   { exact mem_bInter_iff.2 (λ Z hZ, Z.2.2) },
   { rwa [not_nonempty_iff_eq_empty, inter_comm, ←subset_compl_iff_disjoint, compl_compl] at H2 }
 end
+
+section connected_component_setoid
+local attribute [instance] connected_component_setoid
+
+/-- π₀ α is Hausdorff when α is Hausdorff and compact -/
```

Comment:
```suggestion
/-- `π₀ α` is Hausdorff when `α` is Hausdorff and compact -/
```

Reply:
Also: do you want to use `pi_0` in the docstring? Or switch to `connected_components`?

Reply:
No, thats there by mistake